src/TiledArray/cuda/cublas.h

```diff
@@ -76,17 +76,16 @@ class cuBLASHandlePool {
 };
 // thread_local cublasHandle_t *cuBLASHandlePool::handle_;
 
-inline cublasOperation_t to_cublas_op(
-    madness::cblas::CBLAS_TRANSPOSE cblas_op) {
+inline cublasOperation_t to_cublas_op(math::blas::Op cblas_op) {
```

Comment:
Indeed.

This points out the cuBLAS in BLAS++ issue. Having looked at it I don't think there is a way to fully control how the work gets assigned to streams through BLAS++ API, so switching to their device interface and removing all CUDA stuff is not an option as it will break TA contractions. Without looking more deeply I don't see how to work around this.

I propose we defer device BLAS work to another PR.

Reply:
I agree that this can get deferred, but FWIW they do have stream management

https://bitbucket.org/icl/blaspp/src/4cae6e956825496d13e087c4620dd67fe40d75bb/include/blas/device.hh#lines-79

It still needs a bit of work (i.e. they dont allow for attaching a queue onto existing streams, which is something that would be beneficial in this use case)